httpclient5-win/src/main/java/org/apache/hc/client5/http/impl/win/WinHttpClients.java

```diff
@@ -54,14 +52,7 @@ private WinHttpClients() {
     public static boolean isWinAuthAvailable() {
         String os = System.getProperty("os.name");
         os = os != null ? os.toLowerCase(Locale.ROOT) : null;
-        if (os != null && os.contains("windows")) {
-            try {
-                return Sspi.MAX_TOKEN_SIZE > 0;
```

Comment:
This is a good change bacause
* SSPI is always available on Windows
* This Windows module relies on JNA